server/etcdmain/help.go

```diff
@@ -103,6 +105,8 @@ Member:
     Read timeout set on each rafthttp connection
   --raft-write-timeout '` + rafthttp.DefaultConnWriteTimeout.String() + `'
     Write timeout set on each rafthttp connection
+  --feature-gates ''
+    A set of key=value pairs that describe server level feature gates for alpha/experimental features. Options are:'` + strings.Join(features.NewDefaultServerFeatureGate("", nil).KnownFeatures(), ",") + `'
```

Comment:
This is what I got. Please format the output, print each feature in a separate line.
```
  --feature-gates ''
    A set of key=value pairs that describe server level feature gates for alpha/experimental features. Options are:'AllAlpha=true|false (ALPHA - default=false),AllBeta=true|false (BETA - default=false),DistributedTracing=true|false (ALPHA - default=false),StopGRPCServiceOnDefrag=true|false (ALPHA - default=false)'
```

Please let me know if you want to fix it in this PR or a separate PR.

Reply:
updated